### PR TITLE
Update FilterList.json

### PR DESCRIPTION
**Pull request by KonoromiHimaries**
https://github.com/gorhill/uBlock/issues/3406#issuecomment-355748757

+ add gitcdn mirror
+ add githack mirror
+ remove (Obsolete) tag

### CODE REVIEW

File: data/FilterList.json [hunk 1]
```diff
@@ -7,7 +7,6 @@
     "name": "280 Blocker",
     "submissionUrl": "https://docs.google.com/forms/d/e/1FAIpQLScNeZhFrFZt9GhIVGdThGz7oyepcNRKuOi5PJDnsC-awxTeOQ/viewform",
     "syntaxId": 3,
-    "updatedDate": "2018-10-10T08:45:09",
```
Comment (collinbarrett) [inline data/FilterList.json]: Was there a reason you removed this date? It's an auto-updated field when FilterLists senses a list is updated, but can also be specified manually for lists that haven't been updated in a long time (pre-FilterList's scans). Let's keep the date in for now unless you have a reason to remove it. [More Info](https://github.com/collinbarrett/FilterLists/wiki/FilterList)

**Comment by DandelionSprout** [comment]
Welcome to the world of pull requests, Konoromi. ❤️ 

I am uncertain if the `(Obsolete)` tags are truly needed, as it'd by extension mean that more than 100 lists on FilterLists.com would've received that tag.

I've launched a feature request at #598 to have GitCDN and Githack links be automatically created for all of FilterLists.com's ≥500 GitHub/GitLab/Bitbucket lists, but I can see that testing it out for these ~50 lists with this pull could be a good testing ground.

File: data/FilterList.json [hunk 2]
```diff
@@ -2484,7 +2575,7 @@
     "homeUrl": "https://www.facebook.com/adblockpl",
     "issuesUrl": "https://github.com/adblockpolska/Adblock_PL_List/issues",
     "licenseId": 4,
-    "name": "Adblock Polska",
+    "name": "Adblock Polska (Obsolete)",
```
Comment (collinbarrett) [inline data/FilterList.json]: I think we should avoid adding non-title information to the `name` fields as much as possible. Obsolete/Unsupported/etc. duplicates the same info that `updatedDate` notes. We could add a business rule to display something like "Obsolete" in the UI based on the value of `updatedDate` if there was demand for that. For now, let's avoid adding these descriptors to `name` and have any further discussion over in #566

File: data/FilterList.json [hunk 3]
```diff
@@ -42,7 +41,9 @@
     "publishedDate": "2016-11-19T20:26:13",
     "syntaxId": 1,
     "updatedDate": "2016-12-29T17:46:08",
-    "viewUrl": "https://raw.githubusercontent.com/michaeltrimm/hosts-blocking/master/_hosts.txt"
+    "viewUrl": "https://raw.githubusercontent.com/michaeltrimm/hosts-blocking/master/_hosts.txt",
+    "viewUrlMirror1": "https://gitcdn.xyz/repo/michaeltrimm/hosts-blocking/master/_hosts.txt",
```
Comment (collinbarrett) [inline data/FilterList.json]: Might be automated in the future if we implement #598 , but this is great for now.

File: data/FilterList.json [hunk 4]
```diff
@@ -4151,7 +4326,7 @@
     "homeUrl": "https://github.com/AdguardTeam/AdguardFilters",
     "issuesUrl": "https://github.com/AdguardTeam/AdguardFilters/issues",
     "licenseId": 12,
-    "name": "AdGuard Polish Filter (Obsolete)",
+    "name": "AdGuard Polish Filter",
```
Comment (collinbarrett) [inline data/FilterList.json]: Nice. Thanks.

**Pull request merged**